compiler/ballerina-parser/src/test/resources/misc/ambiguity/ambiguity_source_30.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/http;
+
+public function hello() re
+
+
+
+
```

Comment:
Extra newlines?

Reply:
@m36dot Is this intentional?

Reply:
yes, thats exactly the way the source was in the issue